docs/howto/deployment/index.txt

```diff
@@ -22,3 +36,9 @@ they are developed.
    ../static-files/deployment
    ../error-reporting
    checklist
+
+.. _WSGI: https://www.python.org/dev/peps/pep-0333/
```

Comment:
I think that `https://wsgi.readthedocs.io/en/latest/` would be better in this context.